src/components/pages/Upload/UploadCase.js

```diff
@@ -1,98 +1,50 @@
+// import UploadCaseForm from './UploadCaseForm';
+import { notification, Upload, Modal, Button, Spin } from 'antd';
+import {
+  CheckCircleOutlined,
+  CloseCircleOutlined,
+  LoadingOutlined,
+} from '@ant-design/icons';
 import './CaseForm.css';
+import './_UploadCase.less';
 
-const useStyles = makeStyles(theme => ({
-  root: {
-    '& .MuiTextField-root': {
-      margin: theme.spacing(2),
-      width: '30rem',
-      textAlign: 'center',
-    },
-  },
-
-  uploadPage: {
-    display: 'flex',
-    padding: '1%',
-    margin: '0 auto',
-    width: '100%',
-    justifyContent: 'flex-end',
-    marginLeft: '-4rem',
-  },
-
-  pdfUpload: {
-    marginTop: '15%',
-    display: 'flex',
-    marginRight: '7.5%',
-    width: '100%',
-    justifyContent: 'center',
-    flexDirection: 'column',
-  },
-
-  h1Styles: {
-    fontSize: '2rem',
-    marginBottom: '2.5rem',
-  },
-
-  h2Styles: {
-    fontSize: '1.3rem',
-    marginBottom: '2.5rem',
-    width: '100%',
-  },
-
-  buttonStyles: {
-    color: '#ffffff',
-    backgroundColor: '#215589',
-    marginTop: '3%',
-    display: 'flex',
-    alignItems: 'center',
-    justifyContent: 'center',
-  },
-}));
-
-const initialFormValues = {
-  date: '',
-  judge: '',
-  case_outcome: '',
-  country_of_origin: '',
-  protected_grounds: '',
-  application_type: '',
-  case_origin_city: '',
-  case_origin_state: '',
-  gender: '',
-  applicant_language: '',
-  indigenous_group: '',
-  type_of_violence: '',
-  initial_or_appellate: false,
-  filed_in_one_year: false,
-  credible: false,
-};
+// Icons for modal
+import Icon from '@ant-design/icons';
+import UploadCaseBox from '../../../styles/icons/upload-box.svg';
+import OrangeLine from '../../../styles/orange-line.svg';
 
-const HRFBlueLoader = withStyles(() => ({
-  root: {
-    '& .MuiCircularProgress-circle': {
-      color: '#215589',
-    },
-  },
-}))(CircularProgress);
+// const initialFormValues = {
+//   date: '',
+//   judge: '',
+//   case_outcome: '',
+//   country_of_origin: '',
+//   protected_grounds: '',
+//   application_type: '',
+//   case_origin_city: '',
+//   case_origin_state: '',
+//   gender: '',
+//   applicant_language: '',
+//   indigenous_group: '',
+//   type_of_violence: '',
+//   initial_or_appellate: false,
+//   filed_in_one_year: false,
+//   credible: false,
+// };
 
 const UploadCase = ({ authState }) => {
-  const [formValues, setFormValues] = useState(initialFormValues);
+  // const [formValues, setFormValues] = useState(initialFormValues);
   const [formValueQueue, setFormValueQueue] = useState([]);
-  const classes = useStyles();
   const [isLoading, setIsLoading] = useState(false);
   const [isEditing, setIsEditing] = useState(false);
   const { Dragger } = Upload;
   const [postQueue, setPostQueue] = useState([]);
   const [nextPost, setNextPost] = useState(null);
   const [isModalVisible, setIsModalVisible] = useState(false);
+
+  const spinner = <LoadingOutlined style={{ fontSize: 24 }} spin />;
```

Comment:
I like the adding of the loading spinner here, little things like this do a lot for user experience!